src/components/SignupForm/index.tsx

```diff
@@ -172,7 +172,10 @@ export const SignupForm: FC<SignupFormPropType> = ({
             label={
               <>
                 Ich akzeptiere die{" "}
-                <TextLink href='https://www.technologiestiftung-berlin.de/de/datenschutz/'>
+                <TextLink
+                  target='_blank'
```

Comment:
Just for completeness: Can you also add `rel='noopener noreferrer'`?

Reply:
great catch, thx!